src/store/catalog-store.ts

```diff
@@ -21,13 +21,16 @@ import {
 export class CatalogStore {
   public products: Catalog.ProductProjection[] = [];
   public productList: ProductCard[] = [];
+  public categories: Catalog.ProductCategory[] = [];
+  public selectedCategoryId: string = messages.emptyValue;
```

Comment:
нельзя использовать messages как значение. messages - это текст, который мы используем просто как текст

Reply:
```suggestion
  public selectedCategoryId: "";
```
причем вот так будет вполне ок